catalog/dags/maintenance/trim_and_deduplicate_tags.py

```diff
@@ -0,0 +1,88 @@
+"""
+See the issue for context and motivation: https://github.com/WordPress/openverse/issues/4199
+
+This DAG triggers a run of the batched update DAG. It generates a new list of tags by
+trimming all existing tags and re-inserting only the distinct tags of the resulting list of tags.
+"""
+
+from datetime import datetime, timedelta
+from textwrap import dedent
+
+from airflow.decorators import dag
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+
+from common.constants import DAG_DEFAULT_ARGS, MEDIA_TYPES
+from database.batched_update.constants import DAG_ID as BATCHED_UPDATE_DAG_ID
+
+
+DAG_ID = "trim_and_deduplicate_tags"
+
+
+@dag(
+    dag_id=DAG_ID,
+    schedule=None,
+    start_date=datetime(2024, 6, 3),
+    tags=["database"],
+    doc_md=__doc__,
+    max_active_runs=1,
+    default_args=DAG_DEFAULT_ARGS,
+)
+def trim_and_deduplicate_tags():
+    TriggerDagRunOperator.partial(
+        task_id=DAG_ID,
+        trigger_dag_id=BATCHED_UPDATE_DAG_ID,
+        wait_for_completion=True,
+        execution_timeout=timedelta(hours=5),
+        max_active_tis_per_dag=1,
```

Comment:
Since this is iterating through the media tables it's safe to start both at the same time instead of waiting. That could speed up things a bit :)

```suggestion
        max_active_tis_per_dag=2,
```

Reply:
Good thought! Although this is the `max_active_tis_per_dag`, which looks like [according to the docs](https://airflow.apache.org/docs/apache-airflow/stable/faq.html#how-to-improve-dag-performance) is:

> This parameter controls the number of concurrent running task instances across dag_runs per task

I think we also want to increase `max_active_runs` on L27 to `2`:

> Overrides [max_active_runs_per_dag](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#config-core-max-active-runs-per-dag)

Reply:
Why do we need to increase the maximum number of DAG runs? Each run for all media tables is 2 tasks within a single run. I think Krystle's change alone would address that :thinking: Or does the max active dag run somehow also limit the max tasks running within a DAG?

Reply:
Oh forgive me, I confused this for building up two separate DAGs rather than one DAG with two media types. That should be fine!